### PR TITLE
Add basic UI

### PR DESCRIPTION
**Pull request by rotimiazeez**
# OOP school library: Add Basic UI🎉 

> In this project, you will create a form of UI for your school library app. This way it can be invoked as an executable and not something you use in IRB exclusively 💪🏽.

Additional description of the project and its features ➕:

- Create a main.rb file that will serve as your console app entry-point. It should have methods that do the following:
         - List all books.
         - List all people.
         - Create a person (teacher or student, not a plain Person).
         - Create a book.
         - Create a rental.
         - List all rentals for a given person id.
- In your main.rb define the entry point, this will be a method called main that is invoked at the end of your file. This method should do the following:
       - Present the user with a list of options to perform.
       - Lets users choose an option.
       - If needed, ask for parameters for the option.
       - Have a way to quit the app.

## Built With ❤️ 

- Ruby

## Deployed Preview 🚀 
[Deployed Preview](Coming soon)

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+    print 'Age: '
+    age = gets.chomp
+
+    print 'Name: '
+    name = gets.chomp
+
+    print 'Has parent permission ❓ [Y/N]: '
+    parent_permission = gets.chomp.downcase == 'y'
+
+    student = Student.new(name: name, age: age, parent_permission: parent_permission, classroom: @classroom)
```
Comment (henatan99) [inline main.rb:98]: - [ ] This method is throwing an error while I try to create a student. Kindly fix this issue. :pray: 

Screenshot from 2021-11-13 00-34-07

File: main.rb [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+    print 'Age: '
+    age = gets.chomp
+
+    print 'Name: '
+    name = gets.chomp
+
+    print 'Specialization: '
+    specialization = gets.chomp
+
+    teacher = Teacher.new(name: name, age: age, specialization: specialization)
```
Comment (henatan99) [inline main.rb:112]: - [ ] Same issue here. When I try to create a teacher, it throws an argument error. Kindly fix this issue. :pray: 

Screenshot from 2021-11-13 00-38-37

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @rotimiazeez , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
- [ ] Kindly make sure all the functionalities are working as required by the project before submitting for another review. :pray: 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.rb (hunk 1 above)
Comment (rotimiazeez) [reply main.rb:98]: This is working perfectly.
student

File: main.rb (hunk 2 above)
Comment (rotimiazeez) [reply main.rb:112]: This is working perfectly too.
teacher

**Comment by rotimiazeez** [comment]
Hi @henatan99👋,
I don't understand why you are getting this error it's all working well without errors.🤷‍♂️
teacher2
same on Vscode👌

File: main.rb (hunk 1 above)
Comment (rloterh) [reply main.rb:98]: This issue is well resolved! ✔️

File: main.rb (hunk 2 above)
Comment (rloterh) [reply main.rb:112]: This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Azeez, 

Your project is complete! Great job for making the necessary changes recommended in the previous review. There is nothing else to say other than... it's time to merge it 🍾 :shipit:.
Congratulations! 🎉🥂

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**